test/mainline_test.go

```diff
@@ -236,6 +269,7 @@ func setupManager() (client.Client, manager.Manager) {
 	err = controller.AddToManager(mgr, options.AddOptions{
 		DetectedProvider:    operator.ProviderNone,
 		EnterpriseCRDExists: true,
+		AmazonCRDExists:     true,
```

Comment:
Was this changed needed/intended?

Reply:
This fixes an issue with the tests where the tests in mainline_test.go exit early because we hit this line: https://github.com/tigera/operator/blob/master/pkg/controller/installation/core_controller.go#L490

We install all the crds right before the tests are run so we end up hitting that "reboot" and skip the remainder of tests in mainline_test.go